src/web/index.html

```diff
@@ -15,13 +15,13 @@
 
     <div id="live-chart-page">
       <div id="displaying-reading" style="visibility: hidden">Your plant's current soil moisture reading:
-        <span id="reading"></span>
-        <br>
-        <span id="moisture-text"></span>
+        <span id="reading"></span><br>
+        <span id="moisture-text">placeholder text</span>
```

Comment:
Do we need placeholder text?

Reply:
Only to keep the gap when it's initially hidden. If there is another way to do this we can change it.